.github/workflows/ci.yaml

```diff
@@ -767,6 +767,27 @@ jobs:
           set -euxo pipefail
           kubectl --namespace coder rollout restart deployment/coder
           kubectl --namespace coder rollout status deployment/coder
+  
+  deploy-wsproxies:
+    runs-on: ubuntu-latest
+    needs: build
+    if: !github.event.pull_request.head.repo.fork
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup flyctl
+        uses: superfly/flyctl-actions/setup-flyctl@master
+
+      - name: Deploy workspace proxies
```

Comment:
Can we store manifests for these in the repo? In case we need to change proxy env vars perhaps

Reply:
yes we can store. That will require encrypting secrets or domain host names.e.g (`PROXY_TOKEN`)

Reply:
Can those secrets just get stored in a "secrets" tab in fly.io? IDK how fly works so maybe that doesn't exist

Reply:
Yes we can do that too.